### PR TITLE
Dynamic review star rendering

### PR DESCRIPTION
**Pull request by Jvenci**
Hello @RamKhalsa ! 

For a reason I believe I know, this commit is showing the file changes that I sent to Toshi earlier today, I believe it is because I haven't been able to merge that branch yet. 

Please just review the client/src/Stars.jsx file, db seed, and test files. 

Thank you!!

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -6,25 +6,48 @@ class App extends Component {
...
+        next: res.data.slice(3),
         previous: [],
       }))
       .catch(err => console.error('Unable to complete: ', err));
   }
+
   handleNext() {
-    this.state.previous.length === 9 ? 'conditional render arrow away'
+    const { previous, next, currentDisplay } = this.state;
+    const newPrev = [...previous, currentDisplay.shift()];
```
Comment (RamKhalsa) [inline client/src/App.jsx]: Nice use of destructuring here!

File: client/src/Homes.jsx [hunk 2]
```diff
@@ -2,37 +2,71 @@ import React from 'react';
 import './style.css';
 import Stars from './Stars';
 
-const Homes = (props) => {
-  const { data } = props;
-  const listing = data.map((home) => {
+const Homes = ({ handleNext, data, prev, next, handlePrev }) => {
+  const listing = data.map(home => (
```
Comment (RamKhalsa) [inline client/src/Homes.jsx]: may want to de-structure home here--just a thought!

File: client/src/App.jsx (hunk 1 above)
Comment (Jvenci) [reply client/src/App.jsx]: Thank you!!

File: client/src/Homes.jsx (hunk 2 above)
Comment (Jvenci) [reply client/src/Homes.jsx]: nice catch, will implement, thank you!

**Pull request merged**